src/components/loader-bar/loader-bar.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import LoaderBar from ".";
+
+export const DefaultStory: ComponentStory<typeof LoaderBar> = (args) => (
+  <LoaderBar mt={2} {...args} />
+);
+DefaultStory.storyName = "default";
```

Comment:
```suggestion
export const DefaultStory: ComponentStory<typeof LoaderBar> = () => (
  <LoaderBar mt={2} />
);
DefaultStory.storyName = "default";
```
We don't need to pass `args` here